Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.10-slim
 
 # Optional build argument for different environments
 ARG ENV
 
 RUN apt-get update -y \
-    && apt-get install --no-install-recommends -y gcc libc-dev musl-dev postgresql-client libpq-dev
+    && apt-get install --no-install-recommends -y gcc libc-dev musl-dev postgresql-client libpq-dev make
 
-RUN pip install --upgrade pip && pip install pipenv
+RUN pip install --upgrade pip && pip install pipenv==v2022.11.30
```

Comment:
Why did we have to pin `pipenv`?

Reply:
The version of pipenv from the day before had a bug that prevented the lockfile being generated, so when I finally got it to work when the new version came out I decided to pin it. pipenv just doesn't have a great track record WRT bugs, so I decided pinning it is better than having builds start failing at some point in the future.